### DISCUSSION TITLE
Toast docs could be improved, thoughts?

### DISCUSSION THREAD

**Discussion by kylegl** [hngngn/shadcn-solid#108] [General]
Hello, 

First off love this project. Works great. 

I did get stuck on implementing the Toast component because the docs don't show or explain how to implement the `ToastRegion`. Is there any specific reason why you aren't including this in the documentation? The solid-ui project, thats similar to yours, includes the [Toaster component](https://www.solid-ui.com/docs/components/toast#toaster) and explains this is needed to display the toasts. What are your thoughts on including this here and adding relevant info to docs? 

Thanks again for this repo!

**Comment by hngngn** [comment]
Yes, it was in the manual tab. I think I accidentally removed it in the 0.7.4 update. You can refer to the [old](https://shadcn-solid-hvk76im36-hngngns-projects.vercel.app/) version for more information. I will update this soon.
